builder/steps/gen-dockerfile/contents/src/detect_setup.ts

```diff
@@ -28,6 +28,12 @@ import {Logger} from './logger';
 const DEFAULT_APP_YAML = 'app.yaml';
 const PACKAGE_JSON = './package.json';
 const YARN_LOCK = 'yarn.lock';
+const PACKAGE_LOCK_JSON = 'package-lock.json';
+const CANNOT_RESOLVE_PACKAGE_MANAGER = 'The presence of yarn.lock ' +
```

Comment:
nit: `Cannot figure out which package manager to use as yarn.lock and package-lock.json files detected were detected. ...`

Reply:
Fixed